README.md

```diff
@@ -46,6 +47,47 @@ We start from a [Derivative::Rodeo::Manifest::PreProcess](./lib/derivative/rodeo
 
 We process the original manifest in an [Arena](./lib/derivative/rodeo/arena.rb).  During processing we might spawn multiple "child" processes from one derivative.  For example splitting a PDF into one image per page.  Each of those page images would then have their own [Derivative::Rodeo::Manifest::Derived](./lib/derivative/rodeo/manifest/derived.rb) for further processing.
 
+### Inflection Points
+
+There are inflection points that the `Derivative::Rodeo` considers:
+
+1. Spawning processes based on the [MimeType](./lib/derivative/rodeo/type/mime_type.rb)
+2. Spawning processes to [split a PDF](./lib/derivative/rodeo/type/pdf_split_type.rb)
+
+These inflection points conceptually start a new [Chain](./lib/derivative/rodeo/chain.rb) of processing.
+
+*Musing: I write this here, wondering if those inflection points should call out to the public API of Derivative::Rodeo; that way providing a clearer separation of boundary and exposing the inflection as a valid invocation.*
+
+### Configuration
+
+There are two conceptual configuration points:
+
+- [Derivative::Rodeo::Configuration](./lib/derivative/rodeo/configuration.rb) via the [Derivative::Rodeo.config](./lib/derivative/rodeo.rb) method.
+- The individual classes in the Derivative::Rodeo namespace via [ActiveSupport's class_attribute](https://api.rubyonrails.org/classes/Class.html#method-i-class_attribute)).
+
+Let’s consider the following.
+
+For one project I need to have two rodeos.  The first rodeo is for pre-processing.  The second rodeo is for ingesting the pre-processed files (see the [Conceptual Diagram](#conceptual-diagram) section).  The [storage](./lib/derivative/rodeo/storage_adapters) and [queue](./lib/derivative/rodeo/queue_adapters) adapters will be different.  For example, the pre-process local storage will likely be the ingest process’s remote storage.  Both rodeos will likely have the same required steps for processing.
+
+For another project, I will again need two rodeos.  But I want different processing steps; for example I want to add steps to process a 3D model.  I might configure the mime type step to sniff out the files that go into a 3D model and then spawn a new step.
+
+For a third project, I again need two rodeos, but the I want to use a different process to determine the file’s mime type; perhaps instead of leveraging the [Marcel gem](http://rubygems.org/gems/marcel), I leverage [Fits](http://fitstool.org) and some XML parsing.
```

Comment:
```suggestion
For a third project, I again need two rodeos, but then I want to use a different process to determine the file’s mime type; perhaps instead of leveraging the [Marcel gem](http://rubygems.org/gems/marcel), I leverage [Fits](http://fitstool.org) and some XML parsing.
```